docs/pages/architecture/nodes.mdx

```diff
@@ -58,60 +65,89 @@ A Teleport Cluster is a set of one or more machines whose certificates are signe
 
 ## Connecting to Nodes
 
-When a client requests access to a Node, authentication is always performed
-through a cluster proxy. When the proxy server receives a connection request
-from a client it validates the client's credentials with the Auth Service. Once
-the client is authenticated the proxy attempts to connect the client to the
-requested Node.
+Nodes support two modes - a standard mode and a reverse tunnel mode.
+
+In standard mode, nodes act like OpenSSH servers that
+only accept client SSH certificates. Users can connect to nodes through Proxy as a jump-host
+or directly:
 
-There is a detailed walk-through of the steps needed to initiate a connection to
-a node in the [Architecture Overview](overview.mdx).
+![Standard Mode](../../img/architecture/ssh-direct-mode@1.2x.svg)
 
-![Proxy Connection between client and node](../../img/proxy_client_connect.svg)
+In reverse tunnel mode, nodes establish reverse tunnels back to proxy. Nodes
+do not bind to any interface on the host, making sure the connection is only possible
+through proxies:
+
+![Tunnel Mode](../../img/architecture/ssh-tunnel-mode@1.2x.svg)
+
+<Notice type="tip">
+You can mix both modes in the same cluster, depending on your use case.
+For example, you can have several IOT devices joining the cluster via reverse tunnel
+and a large fleet of servers in the internal network using standard mode.
+</Notice>
 
 ## Cluster state
 
 Cluster state is stored in a central storage location configured by the Auth
-Server. This means that each node is completely stateless and holds no secrets
+Server. Each node (or proxy) is stateless and holds no secrets
 such as keys or passwords.
 
-![Cluster State](../../img/cluster_state.svg)
-
-The cluster state information stored includes:
+The cluster state includes:
 
 - Node membership information and online/offline status for each node.
 - List of active sessions.
 - List of locally stored users.
 - RBAC configuration (roles and permissions).
 - Dynamic configuration.
 
-Read more about what is stored in the [Auth Guide](authentication.mdx#auth-state)
-
-## Session recording
+## SSH Session recording
 
 By default, nodes submit SSH session traffic to the Auth server
 for storage. These recorded sessions can be replayed later via `tsh play`
 command or in a web browser.
 
+### SSH node recording
+
 Some Teleport users assume that audit and session recording happen by default
 on the Teleport proxy server. This is not the case in default configuration
 because a proxy cannot see the encrypted traffic, it is encrypted end-to-end,
 i.e. from an SSH client to an SSH server/node, see the diagram below:
 
 ![session-recording-diagram](../../img/session-recording.svg)
 
-However, starting from Teleport 2.4, it is possible to configure the
-Teleport proxy to enable "recording proxy mode".
+### SSH proxy mode 
```

Comment:
Any reason not to call this proxy recording mode? This is how we usually refer to it when speaking to customers or prospects, so it might make searching easier.